test/model.user-identity.test.js

```diff
@@ -187,9 +187,53 @@ describe('UserIdentity', function () {
 
           assert.equal(user.username, 'ldap.xyzldap');
           assert.equal(user.email, 'fooldap@bar.com');
-          
+
           done();
         });
       });
   });
+
+  it('supports accessToken to be true with local accounts', function (done) {
```

Comment:
what does this message mean? describe the problem in a context without details to the implementation